### PR TITLE
colord-gtk: update to 0.3.0.

### PR DESCRIPTION
**Pull request by tibequadorian**
#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/colord-gtk/template [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-revision=2
+version=0.3.0
+revision=1
 build_style=meson
 build_helper=gir
-configure_args="-Dtests=false -Dvapi=$(vopt_if vala true false)
- -Ddocs=$(vopt_if docs true false) -Dman=$(vopt_if man true false)"
+configure_args="-Dtests=false $(vopt_bool vala vapi) $(vopt_bool docs docs)
+ $(vopt_bool man man) $(vopt_bool gir introspection)"
 hostmakedepends="pkg-config intltool glib-devel libxslt
```
Comment (paper42) [inline srcpkgs/colord-gtk/template:9]:
```suggestion
hostmakedepends="pkg-config gettext glib-devel libxslt
```

File: srcpkgs/colord-gtk/template [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
  $(vopt_if docs gtk-doc) $(vopt_if vala vala-devel)
  $(vopt_if man docbook-xsl-ns)"
-makedepends="colord-devel gtk+3-devel"
+makedepends="colord-devel gtk+3-devel gtk4-devel"
 short_desc="GTK support library for colord"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="https://github.com/hughsie/colord-gtk"
```
Comment (paper42) [inline srcpkgs/colord-gtk/template:16]:
```suggestion
homepage="https://github.com/hughsie/colord-gtk"
changelog="https://raw.githubusercontent.com/hughsie/colord-gtk/main/NEWS"
```

**Comment by paper42** [comment]
gir doesn't seem to be optional
```
Found pkg-config: /usr/bin/pkg-config (0.29.2)
Run-time dependency glib-2.0 found: YES 2.70.2
Run-time dependency gio-2.0 found: YES 2.70.2
Run-time dependency colord found: YES 1.4.5
Run-time dependency gtk+-3.0 found: YES 3.24.31
Run-time dependency gtk4 found: YES 4.6.1
Configuring config.h using configuration
Configuring cd-version.h using configuration

libcolord-gtk/meson.build:141:10: ERROR: Unknown variable "libcolord_gtk_gir".

A full log can be found at /builddir/colord-gtk-0.3.0/build/meson-logs/meson-log.txt
```

**Comment by tibequadorian** [comment]
Thank you, I should've noticed this. It seems that it's required when vala is enabled. The build option is probably useless anyway so I removed it now instead of adding additional checks.

File: srcpkgs/colord-gtk/template [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
-checksum=2a4cfae08bc69f000f40374934cd26f4ae86d286ce7de89f1622abc59644c717
+checksum=b9466656d66d9a6ffbc2dd04fa91c8f6af516bf9efaacb69744eec0f56f3c1d0
 
 build_options="vala docs man"
-build_options_default="vala"
-desc_option_docs="Generate documents"
-desc_option_man="Generate manpage"
+build_options_default="vala man"
+desc_option_docs="Generate documentation"
+desc_option_man="Generate man pages"
```
Comment (paper42) [inline srcpkgs/colord-gtk/template:24]: docs should be at least moved to the -devel subpackage

Comment (tibequadorian) [reply srcpkgs/colord-gtk/template]: Makes sense. I enabled docs by default now and I wonder if the build option is necessary at all...
Should I rename it to `gtk_doc` to make it consistent with other packages?
`gtk_doc` is also currently missing a description in options.description btw

**Pull request merged**